### ISSUE TITLE
Problems with creation of first object

### ISSUE THREAD

**Issue by renatoalmeidaoliveira** [irrdnet/irrd#288]
**Describe the bug**
When an person, or new mntner object is created using irrd_submit_email or submit_changes.py, the database referential integrity blocks the creation, i think that's because the admin-c is aperson object that requires an mntner object and the mntner object requires an admin-c and there is a circular reference that goes in conflict with the integrity check

**To Reproduce**

* person file, ( from mail)

` 
From: irrd@localhost
To: root@localhost
Date: Mon, 12 Apr 2010 14:21:26 -0400

person:    AS123-admin
address:   ACME STREET
phone:     123456
e-mail:    AS123-admin@localhost
nic-hdl:   AS123-admin
mnt-by:    AS123-admin
changed:   example@example.com
source:    AUTHDATABASE
override:  password
`

* issued command
`irrd_submit_email --config /home/irrd/irrd.yaml --irrd_pidfile /home/irrd/irrd.pid < person`

* Log output

`
---
Create FAILED: [person] AS123-ADMIN

person:    AS123-admin
address:   ACME STREET
phone:     123456
e-mail:    AS123-admin@localhost
nic-hdl:   AS123-admin
mnt-by:    AS123-admin
changed:   example@example.com
source:    AUTHDATABASE

ERROR: Object AS123-ADMIN referenced in field mnt-by not found in database AUTHDATABASE - must reference mntner.
`

**Expected behaviour**
The object creation

**IRRd version you are running**
irrd-4.0.8

**Additional context**
If there're any other recomended method to include the first instance of the objects please let me know

**Comment by mxsasha** [comment]
Yes, you are right: a mntner must refer to a person, and a person must refer to a mntner. To create a new person/maintainer pair, submit both in the same email. If they are both valid, and authorisation passes, they are both created. Note that if you submit a person/mntner pair, and the mntner is invalid for some reason, the person creation will also fail, so look at any other error messages carefully.

Basically, a single email can contain multiple objects to be created, updated or deleted (whether related or not). In that case, the validity of the changes to each object, takes into account all other changes that are about to be made.
